tools/floodgate/js/floodgate.js

```diff
@@ -31,22 +139,22 @@ async function init() {
 
```

Comment:
`setListeners` should be called after init'ing config, project and projectDetails.

In general I'm not a fan of globals, looks like it's only used for `refreshPage` and  `floodgateContent` so you could just pass directly in to those instead.

Reply:
Makes sense. I have updated the function to consume them as params and have moved the `setListeners` call in `init`. Thanks!